CHANGELOG

```diff
@@ -1,6 +1,8 @@
 Version 0.3.1 (2019-09-26)
 --------------------------
-Update referers.yml to latest version (#75)
+Update referers.yml (#75)
```

Comment:
These updates to the `CHANGELOG` should all be done in the `Prepare for release` commit.